extensions/windows-event-log/tests/CWELTestUtils.h

```diff
@@ -94,3 +94,13 @@ class OutputFormatTestController : public TestController {
   std::string output_format_;
   std::optional<std::string> json_format_;
 };
+
+void generateLogFile(const std::wstring& channel, const std::filesystem::path& path) {
+  HANDLE hEventLog = OpenEventLog(NULL, std::string(channel.begin(), channel.end()).c_str());
```

Comment:
Even though Windows API uses the Hungarian notation for the `HANDLE` object I think we should stick with our style guide in our code for these variables as well, so simply `event_log` would be better.

Reply:
renamed